quasar-kotlin/build.gradle

```diff
@@ -50,7 +50,7 @@ task AnnotationTest(type: Test) {
     classpath = sourceSets.AnnotationTest.runtimeClasspath
 
     jvmArgs = jvmArgs.findAll {!it.contains("javaagent")}
-    jvmArgs "-javaagent:${quasarJar}=a(SUSPENDABLE=Lco/paralleluniverse/kotlin/fibers/MySuspendable;,Lco/paralleluniverse/kotlin/fibers/DennisSuspendable;)"
+    jvmArgs "-javaagent:${quasarJar}=a(SUSPENDABLE=Lco/paralleluniverse/kotlin/fibers/MySuspendable;,co.paralleluniverse.kotlin.fibers.DennisSuspendable)"
```

Comment:
This was discussed with the kernel group a little while ago. It was felt that having more than one type for suspendable would be helpful just in case you have different Cordapps with different Suspendable annotations. For example if in the future we decided to modify suspendable you may want to support more than one.

Reply:
> This was discussed with the kernel group a little while ago. It was felt that having more than one type for suspendable would be helpful just in case you have different Cordapps with different Suspendable annotations. For example if in the future we decided to modify suspendable you may want to support more than one.

*WHY* would we have different CorDapps with different `@Suspendable` annotations when we control the annotation? CorDapp developers are using an API that _we_ provide! I also fail to see how we could modify the `@Suspendable` annotation: Which parameters do you anticipate adding to it, and consequently asking CorDapp developers to set correctly? (Not to mention the need then to modify Quasar to behave differently because of them.)

From my perspective, Quasar needs to support precisely _two_ `@Sispendable` annotations:
- `co.paralleluniverse.fibers.Suspendable` (always - because Quasar itself uses it internally on `Fiber`, `Joinable` etc)
- a custom `@Suspendable` defined by Corda, so that CorDapps don't need to depend on Quasar explicitly.

I am aware of no other requirements.

Reply:
Following discussion this morning I've now made `co.paralleluniverse.fibers.Suspendable` always available. Command line argument now adds to the collection of matched Suspendable annotations.